charts/polkadot-base-services/templates/issuer.yaml

```diff
@@ -1,8 +1,9 @@
 {{ if .Values.createCertIssuer -}}
 apiVersion: cert-manager.io/v1alpha2
-kind: Issuer
+kind: ClusterIssuer
 metadata:
   name: letsencrypt
+  namespace: cert-manager
```

Comment:
What if cert-manager namespace does not exist? Is it better to place it the default namespace ? (even the cloudflare secret)